api/server_test.go

```diff
@@ -45,7 +45,7 @@ func TestServerJSONRPCOveHTTPHandler(t *testing.T) {
 	blockchainAPI := api.NewBlockChainAPI(config, store, mockFlowClient)
 	supportedAPIs := api.SupportedAPIs(blockchainAPI)
 	srv.EnableRPC(supportedAPIs)
-	srv.SetListenAddr("localhost", 8545)
+	srv.SetListenAddr("", 8545)
```

Comment:
I'm curious, did you encounter a problem with `"localhost"`?

Reply:
yes, there was an issue in cloud run
it's a specific requirement for cloud run 
```
The ingress container within an instance must listen for requests on 0.0.0.0 on the port to which requests are sent.
```
https://cloud.google.com/run/docs/container-contract

Reply:
Similar comment as above about the port needing to be an injected variable